lua/plenary/test_harness.lua

```diff
@@ -42,6 +42,8 @@ end
 function harness.test_directory(directory, opts)
   print "Starting..."
   opts = vim.tbl_deep_extend("force", { winopts = { winblend = 3 } }, opts or {})
+  local run_opts = vim.tbl_deep_extend("force", { keep_going = false }, opts.run_opts or {})
+  local run_opts_string = vim.inspect(run_opts):gsub("\n", " ")
```

Comment:
can we do `json_encode` and `json_decode` instead here?

Reply:
Done. Also there was a pretty blatant bug here that I somehow missed when testing (was misusing `run` below instead of replacing with `run_command`), fixed.